middlewares/validators/challenges.js

```diff
@@ -17,6 +17,22 @@ const createChallenge = async (req, res, next) => {
   }
 };
 
+const subscribeToChallenge = async (req, res, next) => {
+  const schema = joi.object().strict().keys({
+    userId: joi.string().required(),
+    challengeId: joi.string().required(),
```

Comment:
for userID and challengeId are we using uuid
if yes then can we have something like this
`Joi.string().guid({version: 'uuidv4'});`  please check the syntax
or is there any regex? please validate that as well